apps/src/code-studio/initApp/project.js

```diff
@@ -826,7 +827,18 @@ var projects = module.exports = {
       pathName += '/' + action;
     }
     return pathName;
-  }
+  },
+
+  saveThumbnail(blob) {
```

Comment:
This method should have a test; it looks like all you'll need to stub is `filesApi.putFile`.